### DISCUSSION TITLE
r30gb - Dome Guard - rebooting

### DISCUSSION THREAD

**Discussion by rpkilgore** [roleoroleo/yi-hack-Allwinner-v2#374] [Q&A] [answered]
I bought 2 brand new Dome Guard cams (r30gb), and when I apply the hack it just constantly reboots.  
So far I have tried to disabled things 1 at a time by changing things to no in the system.comf until I had everything disabled including the web service.  It still reboots unless I wipe everything off the sd card or boot it up without an sd card.  Also tried enabling just the swap file and same results.
I'm attaching a dump of both the hacked cam and the clean cam. to see if that helps. 

I have 2 older versions of the exact cam and the hack works on them, but I did notice some differences in the  mtd.txt.  Attaching the dump from those cams too. 

[backup_DomeGuard_older.zip](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/8129406/backup_DomeGuard_older.zip)
[r30gb_backup_clean.zip](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/8129393/r30gb_backup_clean.zip)
[r30gb_backup_hacked.zip](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/8129394/r30gb_backup_hacked.zip)

Thanks,
Robert

**Comment by roleoroleo** [comment, marked as answer]
New cams are r35gb, not r30gb.
But the hack worked properly, this explains why the cam doesn't reboot when you remove the card.
Try r35gb file.

**Reply by rpkilgore** [reply]
That did the trick.  Thanks!

**Reply by roleoroleo** [reply]
Thank you very much for your donation!

**Reply by rpkilgore** [reply]
No problem.. Thank you for maintaining this repo so well.  That was also for the excellent unbrick tool (saving me from buying another cam) and for the quick answer to my question :)
